interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/merge-reports.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Merge Reports
+---
+# Merge Reports
+
+You may have report pages that do not fit within an entire report template, for instance:
```

Comment:
Change "for instance" to "such as in the following cases"

Reply:
why would you make this change?

Reply:
Because they are providing examples of usage

Reply:
```suggestion
You may have report pages that do not fit within an entire report template in the following cases: